Publication v.0.1.0

eonm-pro opened the pull request:
Pull request pour la publication v.0.1.0 de l'ontologie.

Ajouts :
* de la génération de la documentation de l'ontologie avec Widoco
* des nouveaux contenus pour le site web 
* de la génération des release notes

Corrections :
* du post traitement du profil d'application avec SED. La regex qui permettait de supprimer les liens 404 ne fonctionnait plus.
* des urls du siteweb . Les urls absolues ont été remplacées par des urls relatives, ce qui facilite les tests en localhost

eonm-pro committed:
wip

eonm-pro committed:
Fix: liens cassés

eonm-pro committed:
Fix: urls relatives

eonm-pro committed:
Refactor: refactorisation du Dockerfile

Fix: fix de la regex de sed pour la correction du profil d'application

eonm-pro committed:
Ajout : liens vers l'ontologie

kerphi commented on Dockerfile:
```diff
@@ -1,57 +1,95 @@
+COPY ./siteweb/*   /usr/share/nginx/html/
+COPY ./siteweb/.docker/*   /usr/share/nginx/html/
+
+# Installation des dépendances
```
```suggestion
# Installation des dépendances
# locales : pour avoir les dates en français auto-générées par l'outil pandoc dans footer.html
# pandoc : l'outil pour générer les contenus html du site web à partir des fichiers markdown
# default-jdk : pour pouvoir utiliser l'outil widoco (qui est un outil en java) utilisé pour générer l'ontologie en HTML 
```

pour garder une trace de pourquoi on utilise telle et telle dépendance

kerphi commented on Dockerfile:
```diff
@@ -1,57 +1,95 @@
+      -H "Connection: keep-alive" \
+      -F url='https://docs.google.com/spreadsheets/d/1CZIf3bxuuH3aghFn7B7P89DrLqp_jzv4moI_BpI9PzY/export?format=xlsx' \
+      -F format=text/plain \
+      -o ./rdafr.nt
```
du coup pourquoi avons nous "rdafr.nt" dans le github ici ?
https://github.com/transition-bibliographique/ontologie-rda-fr/blob/main/ontologie/rdafr.nt
pour lever les ambiguïtés, on pourrait le retirer du github vu que sa référence vient du fichier docs.google ?

eonm-pro replied:
C'était pour que @delahousse puisse tester la génération Widoco. Il fallait faire appelle au WS et nettoyer certaines balises avec SED. On peut supprimer le rdafr.nt.

eonm-pro replied:
J'ai supprimé le dossier ontologie : https://github.com/transition-bibliographique/ontologie-rda-fr/pull/8/commits/513a2c7f9140be16d7581e280a7ffa866e182b66

kerphi commented on Dockerfile:
```diff
@@ -1,57 +1,95 @@
+      --template ./template.html \
+      -c style.css \
+      -A ./footer.html \
+      ./release-notes.md -o ./release-notes.html
```
Si je ne me trompes pas on a 3 fichier md : 
- release-notes.md
- 02_CONTENT.md
- 01_INTRO.md

Pour uniformiser on pourrait peut-être les renommer comme ceci ?

- release-notes.md
- index-intro.md
- index-content.md

(tout en minuscule et on ajoute "index" en préfixe pour qu'on comprenne bien que ces deux fichiers permettent d'intervenir sur la page d'accueil)

eonm-pro replied:
ok je m'en occupe

eonm-pro replied:
résolu avec [afbfb0f](https://github.com/transition-bibliographique/ontologie-rda-fr/pull/8/commits/afbfb0f87686e32cdd6780092ddbff07a9e223cb)

eonm-pro committed:
Update Dockerfile

Co-authored-by: Stéphane Gully <gully@abes.fr>

eonm-pro committed:
Fix: uniformisation des noms de fichier

eonm-pro committed:
Suppression du dossier ontologie

Le dossier ontologie n'est plus nécessaire. Il est généré lors de la publication de l'ontologie dans le conteneur Docker.

eonm-pro committed:
Feature : multistage Dockerfile

eonm-pro committed:
Ajout des liens vers l'ontologie

eonm-pro commented:
J'ai mis en place le Dockerfile multistage

eonm-pro commented:
J'ai mis en place le Dockerfile multistage

kerphi reviewed (approved):
Top ! en plus avec du multistage 
Du coup pour moi c'est tout bon tu peux merger histoire et déployer sur https://test.rdafr.fr déjà pour voir ?
